packages/core/src/utils/army.ts

```
@@ -1,312 +1,231 @@
-	}
+export const getTroopName = (troopType: TroopType, troopTier: TroopTier): string => {
+  switch (troopTier) {
+    case TroopTier.T1:
+      switch (troopType) {
+        case TroopType.Knight:
+          return "Footman";
+        case TroopType.Crossbowman:
+          return "Archer";
+        case TroopType.Paladin:
+          return "Horseman";
+      }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)